client/galaxy/scripts/components/JobMetrics/ec2.json

```diff
@@ -0,0 +1,306 @@
+[
```

Comment:
Is there a way to fetch this programmatically?  My concern is that with instances and their pricing changing this is going to be a maintenance burden.

Reply:
https://github.com/hexylena/awstimator

Possible, we can update this once in a while. EU will take care.

Reply:
Sounds good.  Ideally the client just fetches this from an external source on demand, but I'm fine with including it for now -- especially since it's an 'estimate' anyway, and an optional feature.

Reply:
The upstream datasource is gigantic (~1.2Gb), so probably not possible to fetch on demand.

We can update that repo with jenkins or so. Yes, emphasize estimate ;)

Reply:
We can include processing of that huge file in CI and then fetch it from CD as an artifact. (not sure, if CD is possible on circle/github_actions, but I can do it very fast on gitlab CI)

Reply:
Sure, definitely an option!

My feeling is Return on (time) Investment of automating the update of this file is very low, I'm not sure anyone will complain even if the numbers are 1-2 years out of date, since they are not making any decisions based on this estimate, and few people see it.

Reply:
Yeah, to clarify I definitely don't mean fetching gigs of data on demand -- Just the 300 line .json.

Reply:
@dannon Yeah. Probably, next time we need to update it, we can include https://github.com/hexylena/awstimator in github CI (maybe compliment it a little with sorting and deduplication). Then we fetch output using this https://developer.github.com/v3/actions/artifacts/ (hopefully we don't end up with CORS)

Reply:
@OlegZharkov Sounds good to me.  Galaxy has a built-in CORS handling mechanism, wouldn't be a problem hopefully.